src/tracing.h

```diff
@@ -19,7 +19,7 @@
 // so the macros below function correctly
 void ReportWrappedOperation1(const char *, Obj op);
 void ReportWrappedOperation2(const char *, Obj op1, Obj op2);
-void InstallOpWrapper(voidfunc, voidfunc);
+void InstallOpWrapper(void (*activate)(void), void (*deactivate)(void));
```

Comment:
Sorry if this is a stupid question, but does this change have anything to do with the above changes or is it independent?

Reply:
Yes, voidfunc is a typedef in system.h and it felt silly to include that just for this trivial typedef

Reply:
Oh right, thanks for explaining! :)